content/realtime/presence.textile

```diff
@@ -505,6 +506,27 @@ Any time a channel is re-attached and the presence set is re-synced, e.g. after
 
 The exception is if you use the "recover":/realtime/connection#connection-state-recovery feature to resume a previous connection with a fresh client library instance (for example, to have continuity over a page refresh). In that case you will need to explicitly re-enter presence after you re-attach to the channel, due to the loss of client library internal state.
 
+h3(#unstable-connections). Presence with unstable connections 
+
+Clients that are part of a presence set remain present for 15 seconds after they are abruptly disconnected, for example where the internet connection suddenly drops or the client is changing networks. This delay is to avoid repeated @leave@ and @enter@ events being sent when a client is experiencing an unstable connection.
+
+The Ably client library will attempt to "reconnect":/realtime/connection#connection-state-recovery after a disconnect. If the connection is reestablished before 15 seconds have passed, a @leave@ event will not be sent. If the connection is reestablished after 15 seconds, a @leave@ event will be sent and the presence set will need to be rejoined.
+
+Note that the 15 second delay from being removed from the presence set is only for abrupt or unplanned disconnects. If a client calls "@leave()@":#leave or "@close()@":/realtime/connection#close they immediately send a @leave@ event.
+
+The time taken before a @leave@ event is sent in the case of an abrupt disconnect can be reduced to a minimum of 1 second by setting a value for @dormantAfter@, in milliseconds. This property is set within the @transportParams@ property of the "@clientOptions@":/realtime/usage#client-options object.
```

Comment:
We should probably make a decision on what we actually want to call this parameter before publishing our docs on it. It's been pointed out that "dormant" isn't a concept the customer actually knows about. cf https://github.com/ably/realtime/pull/3008#issuecomment-631565738

cc @ben-pickering

Reply:
Updated `dormantAfter` to `remainPresentFor` as per https://github.com/ably/realtime/pull/3869